ext/tidy/tidy.c

```diff
@@ -1073,6 +1077,8 @@ static PHP_MINFO_FUNCTION(tidy)
 	php_info_print_table_header(2, "Tidy support", "enabled");
 #if HAVE_TIDYBUFFIO_H
 	php_info_print_table_row(2, "libTidy Version", (char *)tidyLibraryVersion());
+#elif HAVE_TIDYP_H
+	php_info_print_table_row(2, "libTidy Version", (char *)tidyVersion());
 #endif
 	php_info_print_table_row(2, "libTidy Release", (char *)tidyReleaseDate());
```

Comment:
With libtidyp this still should be ifdefd, as far as i understood?

Thanks.

Reply:
Release date ?

Reply:
I ve tried compiling otherwise and it went ok

Reply:
Yep. It's ifdef'd in the function, too.

Reply:
Ah no in fact the function should not be ifdef ... good catch.

Reply:
Ok, no big issue. I'll check it again then, will add an ifdef after merge, if needed.

Reply:
But it s already the case finally apparently the condition had been removed around the function (it s just inside following your earlier recommendation).

Reply:
Yeah, that's fine. What i mean is the not ifdef'd call to `tidyReleaseDate` in `MINFO`, as it's not available in libtidyp. Only that, the userland function tidy_get_release() is fine.

Thanks

Reply:
Oh right you re definitely right here :-)

Reply:
Updated ... thx for your patience.